app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
+         used to label a set of radio buttons that allow the user to change the user agent (UA) string
+         of the browser. -->
     <string name="developer_options_ua_mode">User Agent Mode</string>
+    <!-- LOCALIZATION NOTE (developer_options_events): FIXME This option should be removed -->
     <string name="developer_options_events">Input Events</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_density">Display Density:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_window_size">Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_mouse">Mouse</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_touch">Touch</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_edit">Edit</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_save">Save</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_display_dpi">Display DPI:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_max_window_size">Max. Window Size:</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa">MSAA</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_disabled">Disabled</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_2">2x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_msaa_4">4x</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_desktop">Desktop</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_mobile">Mobile</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_ua_vr">VR</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset">Reset Developer Settings</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_reset_button">Reset</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_environments">Environment</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_meadow">Meadow</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_cave">Cave</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_env_void">Void</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_color">Pointer Color</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_white">White</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="developer_options_pointer_purple">Purple</string>
+    <!-- LOCALIZATION NOTE ():  -->
     <string name="on">ON</string>
```

Comment:
perhaps the L10n community can inform here, but I _believe_ that we ought to not have stylistic capitalisation (because the capitalised ON/OFF pairing does not necessarily make sense in other languages).